job.go

```diff
@@ -87,10 +96,10 @@ func NewJob() Job {
 	ramMb := aurora.NewResource()
 	diskMb := aurora.NewResource()
 
-	resources := make(map[string]*aurora.Resource)
-	resources["cpu"] = numCpus
-	resources["ram"] = ramMb
-	resources["disk"] = diskMb
+	resources := make(map[ResourceType]*aurora.Resource)
```

Comment:
you can use map literals,which save some 'resources' https://tour.golang.org/moretypes/20

````go
resources := map[ResourceType]*aurora.Resoure{
      CPU: numCpus
}
````

Reply:
Will do!